catalog-rest-service/src/main/resources/ui/src/pages/explore/index.tsx

```diff
@@ -192,7 +234,7 @@ const ExplorePage: React.FC = (): React.ReactElement => {
       searchText,
       0,
       0,
-      getFilterString(filters),
+      emptyValue,
```

Comment:
Why did we remove filter from here?

Reply:
because we don't want to change the count's of other tabs when we select a filter on one tab.